src/guide/profiles.md

```diff
@@ -41,16 +41,18 @@ All profiles are prepended by an `@` and in snake case, while all mutators are i
 Currently, infection supports the following profiles:
 * `@arithmetic`
 * `@boolean`
+* `@equal` - Disabled by default
 * `@cast`
 * `@conditional_boundary`
 * `@conditional_negotiation`
 * `@function_signature`
+* `@identical` - Disabled by default
 * `@number`
 * `@operator`
 * `@regex`
 * `@return_value`
 * `@sort`
 * `@zero_iteration`
-* `@default` - This is the default profile, which currently contains all mutators, and is used if no mutator or profile is chosen
+* `@default` - The standard profile, including all other profiles/mutators unless stated otherwise.
```

Comment:
This could be better... How about if we just change `all` to `most` and leave it at it?